packages/xod-client/src/core/styles/components/PatchDocs.scss

```diff
@@ -42,6 +42,7 @@ $PinInfo-padding-bottom: 10px;
       &.number { color: $color-datatype-number }
       &.string { color: $color-datatype-string }
       &.generic { color: $color-datatype-generic }
+      &.byte { color: $color-datatype-byte }
```

Comment:
Has nothing to do with the PR topic, I think.

Reply:
Yep, but... Okay, I'll fix it in another one :)